R/diagnostics.R

```diff
@@ -19,6 +19,10 @@
 #'   tolerance set by {tol}, FALSE otherwise.
 #' @export
 isTimestepRegular <- function(dates, hist=TRUE, tol=.Machine$double.eps^0.5, handler=stop) {
+  #silly thing needed to pass R CMD check
+  # from http://stackoverflow.com/questions/8096313/no-visible-binding-for-global-variable-note-in-r-cmd-check
+  aes=TimeInterval=geom_histogram=xlab=NULL
```

Comment:
Ok, there are two issues at play here. 1) sometimes roxygen doesn't find elements from other packages which you fixed with the @importFrom instead of my =NULL which I agree that makes sense and is better. 2) sometimes roxygen thinks local variables are global variables which is what is happening I believe with s.hat and maybe TimeInterval but I'm not sure if TimeInterval is a local variable or maybe it comes from somewhere else? Then where does it come from? I think the right thing to do in case 2) is still to do the =NULL trick.